### ISSUE TITLE
Note Tab

### ISSUE THREAD

**Issue by netmender** [ezra-bible-app/ezra-bible-app#761]
It would nice to have a clickable side note tab that shows verses with note. This makes it easy to have overview of notes.

Thank you

**Comment by tobias-klein** [comment]
Thank you for your suggestion @netmender!

To confirm I understand your needs correctly:
- You would like to have a note panel in the side bar that can be toggled like the other features in the side bar (tag list, tag statistics, dictionary).
- The note panel would show nothing by default.
- Once a verse is clicked, the note panel would show the rendered notes of the currently selected verse.
- When clicking on the content of the note panel, the editor would open (similar to the existing note editors).

Please confirm! :)

**Comment by netmender** [comment]
Yes, you are right.

**Comment by netmender** [comment]
Yes, You are right

**Comment by netmender** [comment]
May I ask, do you have any of the following bibles in markdown format.
English Standard Version
New English Translation
New International Version
New American Standard Bible
New Living Translation

Thank you

**Comment by tobias-klein** [comment]
> May I ask, do you have any of the following bibles in markdown format.

No, I do not have Bibles in the markdown format. Note that some of these Bibles you listed are copyright-protected and likely it would not be legal to publish them on GitHub or something similar.

Please ask such questions in the [Discussions](https://github.com/ezra-bible-app/ezra-bible-app/discussions) section in the future!
